src/components/LogInPage.js

```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types'
 import { Button, FormControl, FormGroup } from 'react-bootstrap'
 import { Link } from 'react-router-dom'
 
+const WARNING_STYLE = 'warning'
```

Comment:
you shouldn't be defining constants in more than one place. Define them once and then export/import them.